zbus/src/connection/mod.rs

```diff
@@ -1217,6 +1229,24 @@ impl Connection {
             .map_err(Into::into)
     }
 
+    /// Gracefully close the connection, waiting for all other references to be dropped.
+    ///
+    /// This will not disrupt any incoming or outgoing method calls, and will await their
+    /// completion.
```

Comment:
Great job on documenting this. :+1: 

* It would be good to add a simple sample code too.
* Would be nice to update the `Connection` docs too.